templates/index.html

```diff
@@ -1,29 +1,118 @@
+<script src="{% static 'js/index.js' %}"></script>
+{% endblock %}
+
+{% block main-content %}
+
+  <div class="community-header-image">
+    <img src="{% static 'images/os_community_header.jpg' %}">
+  </div>
+
+  <div class="center about-org custom-green-color-font" id="about" style="margin-top: -2%">
+    <div class="center">
+      <h3>Welcome to {{ org.name }}</h3>
+      <p style="margin: 0% 10%; color: black">{{ org.about }}</p>
+    </div>
+  </div>
+  {% if quote_details %}
+  <div class="center quote-of-day custom-green-color-font">
+    <div class="center apply-flex center-content">
+      <h3 style="padding-right: 15px">~</h3>
+      <div class="quote">
+        <p style="margin-bottom: 0px">{{ quote_details.quote }}</p>
+        <h6 style="margin-top: 0px">- {{ quote_details.author }}</h6>
+      </div>
+      <h3 style="padding-left: 15px">~</h3>
+    </div>
+  </div>
+  {% endif %}{# if quote_details #}
+
+  <div class="center org-teams-stats apply-flex center-content custom-green-color-font">
+    {% for team_name, members_count in org.team_details.items %}
+    <div class="team" style="margin-right: 20px">
+      <h3 style="margin-bottom: 0">{{ members_count }}</h3>
+      <h4 style="margin-top: 0">{{ team_name }}</h4>
+    </div>
+    {% endfor %}{# for team_name, members_count in org.team_details.items #}
+  </div>
+
+  <div class="organization-map">
+    {% include 'map.html' %}
+  </div>
+
+  <div class="top-contributors">
+    <div class="top-meta-reviewers">
+      <table class="highlight centered">
+        <thead class="custom-green-color-font">
+          <tr>
+            <th>Rank</th>
+            <th>Username</th>
+            <th>Meta-review Score</th>
+          </tr>
+        </thead>
+        <tbody>
+        {% for contrib in top_meta_review_users %}
+          <tr>
+            <td>{{ contrib.rank }}</td>
+            <td>{{ contrib.login }}</td>
+            <td>{{ contrib.score|floatformat:0 }}</td>
+          </tr>
+        {% endfor %}{# for contrib in top_meta_review_users #}
+        </tbody>
+      </table>
+      <a href="meta-review/" class="waves-effect waves-light btn leaderboards-btn">Contributors Meta-review</a>
+    </div>
+    <div class="top-gamifiers">
+      <table class="highlight centered">
+        <thead class="custom-green-color-font">
+          <tr>
+            <th>Rank</th>
+            <th>Username</th>
+            <th>Gamification Score</th>
+          </tr>
+        </thead>
+        <tbody>
+        {% for index, contrib in top_gamification_users %}
+          <tr>
+            <td>{{ index|add:"1" }}</td>
+            <td>{{ contrib.username }}</td>
+            <td>{{ contrib.score }}</td>
+          </tr>
+        {% endfor %}{# for index, contrib in top_gamification_users #}
+        </tbody>
+      </table>
+      <a href="gamification/" class="waves-effect waves-light btn leaderboards-btn">Gamification leaderboard</a>
```

Comment:
Can you make the whole card clickable instead of just the footer?

I also think that these cards should look similar(in color, etc.) with the individual cards on their respective pages. What's your opinion?

Reply:
Yeas! I will make the whole card clickable. What will make it more good is that we can add a `href` to each row which when clicked will redirect the user to the same user-details card on which the user has clicked.
> Can you make the whole card clickable instead of just the footer?

I tried to give them same color (all green color variants being used in website) but it wasn't looking good, it was giving a dull-like experience. Currently, both of them are having that background colors which are going in match with header, body-background as well as text. Also, the yellow-orange color combination is one of the famous combinations.
>I also think that these cards should look similar(in color, etc.) 

We can surely do that! but the only issue is space, we can't separate the current main-content div into two `div blocks` in which one `div` block will be used for a single card. Also, if we don't choose to create partitions then the concern is where will it be shown at top or bottom of page. Neither of the positions will have a nice UX experience. Showing at home-page will give user a glance giving a nice UX experience and with giving them an additional information that the website has two types of leaderboards related to contribs. And, a new-user to website always recognizes the website by its homepage and easy-navigability. 
> with the individual cards on their respective pages. What's your opinion?